app/styles/app.css

```diff
@@ -115,3 +115,26 @@ a.sub-navigation:hover {
 	color: orange;
 	text-decoration: none;
 }
+.list-frame {
```

Comment:
I would suggest that component-specific CSS styles should have a unique name prefix to distinguish them from things intended for use in the overall application.

It seems like components would be more reusable if the styles could be packaged inside the component (not the application) but prefixes would at least help.

In the future there's an addon for this purpose, but I don't know much about it. https://github.com/ebryn/ember-component-css